src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,61 @@
+
+package calculator
+
+class Calculator {
+    fun calculate(input: String?): Int {
+        if (input.isNullOrBlank()) throw IllegalArgumentException("Invalid input")
+        val inputs = input.split(" ")
+        val queue: ArrayList<String> = arrayListOf()
+        return processStrings(inputs, queue)
+    }
+
+    private fun processStrings(
+        inputs: List<String>,
+        queue: ArrayList<String>,
+    ): Int {
+        for (i in 0 until inputs.size) {
+            addQueue(inputs[i], queue)
+            processQueue(queue)
+        }
+        return queue[0].toInt()
+    }
```

Comment:
```suggestion
    private fun processStrings(
        inputs: List<String>
    ): Int {
        val queue = MutableListOf()
        for (i in 0 until inputs.size) {
            addQueue(inputs[i], queue)
            processQueue(queue)
        }
        return queue[0].toInt()
    }
```

코틀린은 가변과 불변이 명확한 언어에요. 그러기 때문에 List에 대해서도 불변을 List로 표현하고, 가변을 MutableList로 표현을 하죠. 
그러기 때문에, 가변에 대한 속성을 정의할 때는 ArrayList 보다는 MutableList를 사용하고 있습니다. 

그리고, 현재 메서드는 Int로 반환을 하고 있어요. Int를 반환하게 만드는 Queue는 외부에서 받은 queue를 이용하여 함수 내부적으로 계속 추가해 주고 있죠. 외부에서는 queue에 대해서 다루지 않아 보이는데, queue를 processStrings 내부로 옮기면 더 함수가 정리정돈 되며, 외부에서의 변경에도 영향이 없어 보입니다. 

한번 고민하고 변경해 보시길 바랍니다.